subgraph/subgraph.yaml

```diff
@@ -1,17 +1,16 @@
-specVersion: 0.0.4
+specVersion: 0.0.5
```

Comment:
Should we remove that file from git completely?

Reply:
I have removed, this snuck in while trying to fix the tests.